docs/cookbook/presigned-get-download-from-browser.md

```diff
@@ -0,0 +1,51 @@
+# Using pre-signed URLs to download via the browser [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/minio/minio?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+
+Instead of using an S3-compatible client to download files, you can pre-sign a
+request to download a specified file from the browser, even if its bucket is
+private. The following is an annotated example of how this can be used in a
+Node.js application, using [minio-js](https://github.com/minio/minio-js).
+
+This application will work out of the box, just copy each piece together into a
+file and run it.
+
+## Code
+
+```javascript
+const Minio = require('minio')
+
+var client = new Minio({
+    endPoint: 'play.minio.io',
+    port: 9000,
+    secure: true,
+    accessKey: 'Q3AM3UQ867SPQQA43P2F',
+    secretKey: 'zuf+tfteSlswRu7BJ86wekitnifILbZam1KYY3TG'
+})
+```
+
+In order to sign the request, we need to log in to the database. This
```

Comment:
it does not login to minio, it initializes minio client with the creds which will be used to sign the every request